ctfd container fails to load when 4 workers

**Environment**:

  - CTFd Version/Commit: 38416ce5c6e154aa06613e624b17d04761200cbb (Sun May 20 00:25:17 2018 -0400)
  - Operating System: Linux Mint 18.3 
  - Web Browser and Version: Firefox 60.0.1

**What happened?**

When I use multiple workers, my docker container fails to load.
Note that I have a `SECRET_KEY`.
And the issue is the same if I use `.ctfd_secret_key` file instead.

**What did you expect to happen?**

I expected the container to run without any problem :)

**How to reproduce your issue**

1. Clone repository
2. In `docker-compose.yml`,  set this for the ctfd service environment:

```
- SECRET_KEY=219812saknqanqi
- WORKERS=4
```
3. Ensure you have no previous CTFd config/container running. (kill containers, rm -r .data etc)
4. `docker-compose up -d`

**Any associated stack traces or error logs**

When the container fails to load, the logs report:

```
$ docker logs ctfd_ctfd_1
...
Starting CTFd
Traceback (most recent call last):
  File "/usr/local/bin/gunicorn", line 11, in <module>
    sys.exit(run())
  File "/usr/local/lib/python2.7/site-packages/gunicorn/app/wsgiapp.py", line 74, in run
    WSGIApplication("%(prog)s [OPTIONS] [APP_MODULE]").run()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/app/base.py", line 203, in run
    super(Application, self).run()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/app/base.py", line 72, in run
    Arbiter(self).run()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/arbiter.py", line 231, in run
    self.halt(reason=inst.reason, exit_status=inst.exit_status)
  File "/usr/local/lib/python2.7/site-packages/gunicorn/arbiter.py", line 344, in halt
    self.stop()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/arbiter.py", line 393, in stop
    time.sleep(0.1)
  File "/usr/local/lib/python2.7/site-packages/gunicorn/arbiter.py", line 244, in handle_chld
    self.reap_workers()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/arbiter.py", line 524, in reap_workers
    raise HaltServer(reason, self.WORKER_BOOT_ERROR)
gunicorn.errors.HaltServer: <HaltServer 'Worker failed to boot.' 3>
```

>I just tried your commands and it appears to work properly for me

Have you set `WORKERS` to something greater than 1 (e.g 4)? Your diff does not show the `WORKERS`.... if you run it with `WORKERS=1`, you won't have the issue.
And have you started in a clean directory (no .data, no .ctfd_secret_key, no prior running ctfd containers)?

>Try running without Docker so you can get a better idea of what the issues are.

The issue does not occur without Docker.

I started ran the command manually in the Docker container to see what's happening.

1. Modified `docker-entrypoint.sh` not to launch gunicorn, but instead just a dummy server to wait and do nothing more. That's my entrypoint personally:

```
diff --git a/docker-entrypoint.sh b/docker-entrypoint.sh
index 4b1db4c..26333e9 100755
--- a/docker-entrypoint.sh
+++ b/docker-entrypoint.sh
@@ -30,10 +30,12 @@ if [ -z "$WORKERS" ]; then
 fi
 
 # Start CTFd
-echo "Starting CTFd"
-gunicorn 'CTFd:create_app()' \
-    --bind '0.0.0.0:8000' \
-    --workers $WORKERS \
-    --worker-class 'gevent' \
-    --access-logfile "${LOG_FOLDER:-/opt/CTFd/CTFd/logs}/access.log" \
-    --error-logfile "${LOG_FOLDER:-/opt/CTFd/CTFd/logs}/error.log"
+#echo "Starting CTFd"
+# gunicorn 'CTFd:create_app()' \
+#     --bind '0.0.0.0:8000' \
+#     --workers $WORKERS \
+#     --worker-class 'gevent' \
+#     --access-logfile "${LOG_FOLDER:-/opt/CTFd/CTFd/logs}/access.log" \
+#     --error-logfile "${LOG_FOLDER:-/opt/CTFd/CTFd/logs}/error.log"
+python -m SimpleHTTPServer 11000
```

2. Launch the container `docker-compose up -d`
3. Attach to the container: `docker exec -it ctfd_ctfd_1 /bin/sh`
4. Launch manually the command: `gunicorn 'CTFd:create_app()' --bind '0.0.0.0:8000' --workers 4`

It's strange. Because **the first time, it fails**. It complains about a table ctfd.config not existing and then gives that generic `WORKER_BOOT_ERROR`.
The second time, it works (as if there's some missing init that the 1st launch fixes).
Does this ring a bell?

```
ProgrammingError: (pymysql.err.ProgrammingError) (1146, u"Table 'ctfd.config' doesn't exist") [SQL: u'SELECT config.id AS config_id, config.`key` AS config_key, config.value AS config_value \nFROM config \nWHERE config.`key` = %(key_1)s \n LIMIT %(param_1)s'] [parameters: {u'key_1': 'next_update_check', u'param_1': 1}]
```

**Full log**

You see here the first launch (which fails):

```
# gunicorn 'CTFd:create_app()' --bind '0.0.0.0:8000' --workers 4
[2018-05-31 07:36:30 +0000] [41] [INFO] Starting gunicorn 19.7.1
[2018-05-31 07:36:30 +0000] [41] [INFO] Listening at: http://0.0.0.0:8000 (41)
[2018-05-31 07:36:30 +0000] [41] [INFO] Using worker: sync
[2018-05-31 07:36:30 +0000] [51] [INFO] Booting worker with pid: 51
[2018-05-31 07:36:30 +0000] [54] [INFO] Booting worker with pid: 54
[2018-05-31 07:36:30 +0000] [77] [INFO] Booting worker with pid: 77
[2018-05-31 07:36:30 +0000] [78] [INFO] Booting worker with pid: 78
[2018-05-31 07:36:30 +0000] [54] [ERROR] Exception in worker process
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/site-packages/gunicorn/arbiter.py", line 578, in spawn_worker
    worker.init_process()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/workers/base.py", line 126, in init_process
    self.load_wsgi()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/workers/base.py", line 135, in load_wsgi
    self.wsgi = self.app.wsgi()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/app/base.py", line 67, in wsgi
    self.callable = self.load()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/app/wsgiapp.py", line 65, in load
    return self.load_wsgiapp()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/app/wsgiapp.py", line 52, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/usr/local/lib/python2.7/site-packages/gunicorn/util.py", line 364, in import_app
    app = eval(obj, vars(mod))
  File "<string>", line 1, in <module>
  File "/opt/CTFd/CTFd/__init__.py", line 142, in create_app
    update_check(force=True)
  File "/opt/CTFd/CTFd/utils/__init__.py", line 750, in update_check
    next_update_check = get_config('next_update_check') or 0
  File "/usr/local/lib/python2.7/site-packages/flask_caching/__init__.py", line 674, in decorated_function
    rv = f(*args, **kwargs)
  File "/opt/CTFd/CTFd/utils/__init__.py", line 538, in get_config
    config = Config.query.filter_by(key=key).first()
  File "/usr/local/lib/python2.7/site-packages/sqlalchemy/orm/query.py", line 2778, in first
    ret = list(self[0:1])
  File "/usr/local/lib/python2.7/site-packages/sqlalchemy/orm/query.py", line 2570, in __getitem__
    return list(res)
  File "/usr/local/lib/python2.7/site-packages/sqlalchemy/orm/query.py", line 2878, in __iter__
    return self._execute_and_instances(context)
  File "/usr/local/lib/python2.7/site-packages/sqlalchemy/orm/query.py", line 2901, in _execute_and_instances
    result = conn.execute(querycontext.statement, self._params)
  File "/usr/local/lib/python2.7/site-packages/sqlalchemy/engine/base.py", line 948, in execute
    return meth(self, multiparams, params)
  File "/usr/local/lib/python2.7/site-packages/sqlalchemy/sql/elements.py", line 269, in _execute_on_connection
    return connection._execute_clauseelement(self, multiparams, params)
  File "/usr/local/lib/python2.7/site-packages/sqlalchemy/engine/base.py", line 1060, in _execute_clauseelement
    compiled_sql, distilled_params
  File "/usr/local/lib/python2.7/site-packages/sqlalchemy/engine/base.py", line 1200, in _execute_context
    context)
  File "/usr/local/lib/python2.7/site-packages/sqlalchemy/engine/base.py", line 1413, in _handle_dbapi_exception
    exc_info
  File "/usr/local/lib/python2.7/site-packages/sqlalchemy/util/compat.py", line 203, in raise_from_cause
    reraise(type(exception), exception, tb=exc_tb, cause=cause)
  File "/usr/local/lib/python2.7/site-packages/sqlalchemy/engine/base.py", line 1193, in _execute_context
    context)
  File "/usr/local/lib/python2.7/site-packages/sqlalchemy/engine/default.py", line 507, in do_execute
    cursor.execute(statement, parameters)
  File "/usr/local/lib/python2.7/site-packages/pymysql/cursors.py", line 165, in execute
    result = self._query(query)
  File "/usr/local/lib/python2.7/site-packages/pymysql/cursors.py", line 321, in _query
    conn.query(q)
  File "/usr/local/lib/python2.7/site-packages/pymysql/connections.py", line 860, in query
    self._affected_rows = self._read_query_result(unbuffered=unbuffered)
  File "/usr/local/lib/python2.7/site-packages/pymysql/connections.py", line 1061, in _read_query_result
    result.read()
  File "/usr/local/lib/python2.7/site-packages/pymysql/connections.py", line 1349, in read
    first_packet = self.connection._read_packet()
  File "/usr/local/lib/python2.7/site-packages/pymysql/connections.py", line 1018, in _read_packet
    packet.check_error()
  File "/usr/local/lib/python2.7/site-packages/pymysql/connections.py", line 384, in check_error
    err.raise_mysql_exception(self._data)
  File "/usr/local/lib/python2.7/site-packages/pymysql/err.py", line 107, in raise_mysql_exception
    raise errorclass(errno, errval)
ProgrammingError: (pymysql.err.ProgrammingError) (1146, u"Table 'ctfd.config' doesn't exist") [SQL: u'SELECT config.id AS config_id, config.`key` AS config_key, config.value AS config_value \nFROM config \nWHERE config.`key` = %(key_1)s \n LIMIT %(param_1)s'] [parameters: {u'param_1': 1, u'key_1': 'next_update_check'}] (Background on this error at: http://sqlalche.me/e/f405)
[2018-05-31 07:36:30 +0000] [54] [INFO] Worker exiting (pid: 54)
[2018-05-31 07:36:30 +0000] [77] [ERROR] Exception in worker process
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/site-packages/gunicorn/arbiter.py", line 578, in spawn_worker
    worker.init_process()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/workers/base.py", line 126, in init_process
    self.load_wsgi()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/workers/base.py", line 135, in load_wsgi
    self.wsgi = self.app.wsgi()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/app/base.py", line 67, in wsgi
    self.callable = self.load()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/app/wsgiapp.py", line 65, in load
    return self.load_wsgiapp()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/app/wsgiapp.py", line 52, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/usr/local/lib/python2.7/site-packages/gunicorn/util.py", line 364, in import_app
    app = eval(obj, vars(mod))
  File "<string>", line 1, in <module>
  File "/opt/CTFd/CTFd/__init__.py", line 142, in create_app
    update_check(force=True)
  File "/opt/CTFd/CTFd/utils/__init__.py", line 750, in update_check
    next_update_check = get_config('next_update_check') or 0
  File "/usr/local/lib/python2.7/site-packages/flask_caching/__init__.py", line 674, in decorated_function
    rv = f(*args, **kwargs)
  File "/opt/CTFd/CTFd/utils/__init__.py", line 538, in get_config
    config = Config.query.filter_by(key=key).first()
  File "/usr/local/lib/python2.7/site-packages/sqlalchemy/orm/query.py", line 2778, in first
    ret = list(self[0:1])
  File "/usr/local/lib/python2.7/site-packages/sqlalchemy/orm/query.py", line 2570, in __getitem__
    return list(res)
  File "/usr/local/lib/python2.7/site-packages/sqlalchemy/orm/query.py", line 2878, in __iter__
    return self._execute_and_instances(context)
  File "/usr/local/lib/python2.7/site-packages/sqlalchemy/orm/query.py", line 2901, in _execute_and_instances
    result = conn.execute(querycontext.statement, self._params)
  File "/usr/local/lib/python2.7/site-packages/sqlalchemy/engine/base.py", line 948, in execute
    return meth(self, multiparams, params)
  File "/usr/local/lib/python2.7/site-packages/sqlalchemy/sql/elements.py", line 269, in _execute_on_connection
    return connection._execute_clauseelement(self, multiparams, params)
  File "/usr/local/lib/python2.7/site-packages/sqlalchemy/engine/base.py", line 1060, in _execute_clauseelement
    compiled_sql, distilled_params
  File "/usr/local/lib/python2.7/site-packages/sqlalchemy/engine/base.py", line 1200, in _execute_context
    context)
  File "/usr/local/lib/python2.7/site-packages/sqlalchemy/engine/base.py", line 1413, in _handle_dbapi_exception
    exc_info
  File "/usr/local/lib/python2.7/site-packages/sqlalchemy/util/compat.py", line 203, in raise_from_cause
    reraise(type(exception), exception, tb=exc_tb, cause=cause)
  File "/usr/local/lib/python2.7/site-packages/sqlalchemy/engine/base.py", line 1193, in _execute_context
    context)
  File "/usr/local/lib/python2.7/site-packages/sqlalchemy/engine/default.py", line 507, in do_execute
    cursor.execute(statement, parameters)
  File "/usr/local/lib/python2.7/site-packages/pymysql/cursors.py", line 165, in execute
    result = self._query(query)
  File "/usr/local/lib/python2.7/site-packages/pymysql/cursors.py", line 321, in _query
    conn.query(q)
  File "/usr/local/lib/python2.7/site-packages/pymysql/connections.py", line 860, in query
    self._affected_rows = self._read_query_result(unbuffered=unbuffered)
  File "/usr/local/lib/python2.7/site-packages/pymysql/connections.py", line 1061, in _read_query_result
    result.read()
  File "/usr/local/lib/python2.7/site-packages/pymysql/connections.py", line 1349, in read
    first_packet = self.connection._read_packet()
  File "/usr/local/lib/python2.7/site-packages/pymysql/connections.py", line 1018, in _read_packet
    packet.check_error()
  File "/usr/local/lib/python2.7/site-packages/pymysql/connections.py", line 384, in check_error
    err.raise_mysql_exception(self._data)
  File "/usr/local/lib/python2.7/site-packages/pymysql/err.py", line 107, in raise_mysql_exception
    raise errorclass(errno, errval)
ProgrammingError: (pymysql.err.ProgrammingError) (1146, u"Table 'ctfd.config' doesn't exist") [SQL: u'SELECT config.id AS config_id, config.`key` AS config_key, config.value AS config_value \nFROM config \nWHERE config.`key` = %(key_1)s \n LIMIT %(param_1)s'] [parameters: {u'key_1': 'next_update_check', u'param_1': 1}] (Background on this error at: http://sqlalche.me/e/f405)
[2018-05-31 07:36:30 +0000] [77] [INFO] Worker exiting (pid: 77)
[2018-05-31 07:36:30 +0000] [78] [ERROR] Exception in worker process
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/site-packages/gunicorn/arbiter.py", line 578, in spawn_worker
    worker.init_process()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/workers/base.py", line 126, in init_process
    self.load_wsgi()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/workers/base.py", line 135, in load_wsgi
    self.wsgi = self.app.wsgi()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/app/base.py", line 67, in wsgi
    self.callable = self.load()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/app/wsgiapp.py", line 65, in load
    return self.load_wsgiapp()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/app/wsgiapp.py", line 52, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/usr/local/lib/python2.7/site-packages/gunicorn/util.py", line 364, in import_app
    app = eval(obj, vars(mod))
  File "<string>", line 1, in <module>
  File "/opt/CTFd/CTFd/__init__.py", line 142, in create_app
    update_check(force=True)
  File "/opt/CTFd/CTFd/utils/__init__.py", line 750, in update_check
    next_update_check = get_config('next_update_check') or 0
  File "/usr/local/lib/python2.7/site-packages/flask_caching/__init__.py", line 674, in decorated_function
    rv = f(*args, **kwargs)
  File "/opt/CTFd/CTFd/utils/__init__.py", line 538, in get_config
    config = Config.query.filter_by(key=key).first()
  File "/usr/local/lib/python2.7/site-packages/sqlalchemy/orm/query.py", line 2778, in first
    ret = list(self[0:1])
  File "/usr/local/lib/python2.7/site-packages/sqlalchemy/orm/query.py", line 2570, in __getitem__
    return list(res)
  File "/usr/local/lib/python2.7/site-packages/sqlalchemy/orm/query.py", line 2878, in __iter__
    return self._execute_and_instances(context)
  File "/usr/local/lib/python2.7/site-packages/sqlalchemy/orm/query.py", line 2901, in _execute_and_instances
    result = conn.execute(querycontext.statement, self._params)
  File "/usr/local/lib/python2.7/site-packages/sqlalchemy/engine/base.py", line 948, in execute
    return meth(self, multiparams, params)
  File "/usr/local/lib/python2.7/site-packages/sqlalchemy/sql/elements.py", line 269, in _execute_on_connection
    return connection._execute_clauseelement(self, multiparams, params)
  File "/usr/local/lib/python2.7/site-packages/sqlalchemy/engine/base.py", line 1060, in _execute_clauseelement
    compiled_sql, distilled_params
  File "/usr/local/lib/python2.7/site-packages/sqlalchemy/engine/base.py", line 1200, in _execute_context
    context)
  File "/usr/local/lib/python2.7/site-packages/sqlalchemy/engine/base.py", line 1413, in _handle_dbapi_exception
    exc_info
  File "/usr/local/lib/python2.7/site-packages/sqlalchemy/util/compat.py", line 203, in raise_from_cause
    reraise(type(exception), exception, tb=exc_tb, cause=cause)
  File "/usr/local/lib/python2.7/site-packages/sqlalchemy/engine/base.py", line 1193, in _execute_context
    context)
  File "/usr/local/lib/python2.7/site-packages/sqlalchemy/engine/default.py", line 507, in do_execute
    cursor.execute(statement, parameters)
  File "/usr/local/lib/python2.7/site-packages/pymysql/cursors.py", line 165, in execute
    result = self._query(query)
  File "/usr/local/lib/python2.7/site-packages/pymysql/cursors.py", line 321, in _query
    conn.query(q)
  File "/usr/local/lib/python2.7/site-packages/pymysql/connections.py", line 860, in query
    self._affected_rows = self._read_query_result(unbuffered=unbuffered)
  File "/usr/local/lib/python2.7/site-packages/pymysql/connections.py", line 1061, in _read_query_result
    result.read()
  File "/usr/local/lib/python2.7/site-packages/pymysql/connections.py", line 1349, in read
    first_packet = self.connection._read_packet()
  File "/usr/local/lib/python2.7/site-packages/pymysql/connections.py", line 1018, in _read_packet
    packet.check_error()
  File "/usr/local/lib/python2.7/site-packages/pymysql/connections.py", line 384, in check_error
    err.raise_mysql_exception(self._data)
  File "/usr/local/lib/python2.7/site-packages/pymysql/err.py", line 107, in raise_mysql_exception
    raise errorclass(errno, errval)
ProgrammingError: (pymysql.err.ProgrammingError) (1146, u"Table 'ctfd.config' doesn't exist") [SQL: u'SELECT config.id AS config_id, config.`key` AS config_key, config.value AS config_value \nFROM config \nWHERE config.`key` = %(key_1)s \n LIMIT %(param_1)s'] [parameters: {u'key_1': 'next_update_check', u'param_1': 1}] (Background on this error at: http://sqlalche.me/e/f405)
[2018-05-31 07:36:30 +0000] [78] [INFO] Worker exiting (pid: 78)
Traceback (most recent call last):
  File "/usr/local/bin/gunicorn", line 11, in <module>
    sys.exit(run())
  File "/usr/local/lib/python2.7/site-packages/gunicorn/app/wsgiapp.py", line 74, in run
    WSGIApplication("%(prog)s [OPTIONS] [APP_MODULE]").run()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/app/base.py", line 203, in run
    super(Application, self).run()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/app/base.py", line 72, in run
    Arbiter(self).run()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/arbiter.py", line 231, in run
    self.halt(reason=inst.reason, exit_status=inst.exit_status)
  File "/usr/local/lib/python2.7/site-packages/gunicorn/arbiter.py", line 344, in halt
    self.stop()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/arbiter.py", line 393, in stop
    time.sleep(0.1)
  File "/usr/local/lib/python2.7/site-packages/gunicorn/arbiter.py", line 244, in handle_chld
    self.reap_workers()
  File "/usr/local/lib/python2.7/site-packages/gunicorn/arbiter.py", line 524, in reap_workers
    raise HaltServer(reason, self.WORKER_BOOT_ERROR)
gunicorn.errors.HaltServer: <HaltServer 'Worker failed to boot.' 3>
```

Those two lines appear just after. 

```
/opt/CTFd #  * Loaded module, <module 'CTFd.plugins.keys' from '/opt/CTFd/CTFd/plugins/keys/__init__.pyc'>
 * Loaded module, <module 'CTFd.plugins.challenges' from '/opt/CTFd/CTFd/plugins/challenges/__init__.pyc'>
[2018-05-31 07:36:37 +0000] [51] [INFO] Worker exiting (pid: 51)
```

**I launch a second time, and this time it works:**

```
/opt/CTFd # gunicorn 'CTFd:create_app()' --bind '0.0.0.0:8000' --workers 4
[2018-05-31 07:37:15 +0000] [103] [INFO] Starting gunicorn 19.7.1
[2018-05-31 07:37:15 +0000] [103] [INFO] Listening at: http://0.0.0.0:8000 (103)
[2018-05-31 07:37:15 +0000] [103] [INFO] Using worker: sync
[2018-05-31 07:37:15 +0000] [113] [INFO] Booting worker with pid: 113
[2018-05-31 07:37:15 +0000] [114] [INFO] Booting worker with pid: 114
[2018-05-31 07:37:15 +0000] [125] [INFO] Booting worker with pid: 125
[2018-05-31 07:37:15 +0000] [141] [INFO] Booting worker with pid: 141
 * Loaded module, <module 'CTFd.plugins.keys' from '/opt/CTFd/CTFd/plugins/keys/__init__.pyc'>
 * Loaded module, <module 'CTFd.plugins.challenges' from '/opt/CTFd/CTFd/plugins/challenges/__init__.pyc'>
 * Loaded module, <module 'CTFd.plugins.keys' from '/opt/CTFd/CTFd/plugins/keys/__init__.pyc'>
 * Loaded module, <module 'CTFd.plugins.challenges' from '/opt/CTFd/CTFd/plugins/challenges/__init__.pyc'>
 * Loaded module, <module 'CTFd.plugins.keys' from '/opt/CTFd/CTFd/plugins/keys/__init__.pyc'>
 * Loaded module, <module 'CTFd.plugins.challenges' from '/opt/CTFd/CTFd/plugins/challenges/__init__.pyc'>
 * Loaded module, <module 'CTFd.plugins.keys' from '/opt/CTFd/CTFd/plugins/keys/__init__.pyc'>
 * Loaded module, <module 'CTFd.plugins.challenges' from '/opt/CTFd/CTFd/plugins/challenges/__init__.pyc'>
```

--- a/docker-entrypoint.sh
+++ b/docker-entrypoint.sh
@@ -25,6 +25,9 @@ if [ -n "$DATABASE_URL" ]
     sleep 1;
 fi
 
+# Initialize database
+python manage.py db upgrade
+
 if [ -z "$WORKERS" ]; then
     WORKERS=1
 fi

